api/server/docker.go

```diff
@@ -882,6 +887,15 @@ func (d *driver) unmount(w http.ResponseWriter, r *http.Request) {
 	opts := make(map[string]string)
 	opts[options.OptionsDeleteAfterUnmount] = "true"
 
+	errWithoutID := v.Unmount(correlation.TODO(), id, mountpointWithoutID, opts)
```

Comment:
Can you add a comment here explaining why we are doing this double Unmount?

Reply:
Sure, will do that